packages/camera/camera_android_camerax/lib/src/camera_info.dart

```diff
@@ -50,6 +58,21 @@ class CameraInfoHostApiImpl extends CameraInfoHostApi {
         instanceManager.getIdentifier(instance)!);
     return sensorRotationDegrees;
   }
+
+  /// Gets the [LiveData<CameraState>] that represents the state of the camera
+  /// to which the CameraInfo [instance] pertains.
+  Future<LiveData<CameraState>> getLiveCameraStateFromInstance(
+      CameraInfo instance) async {
+    final int? identifier = instanceManager.getIdentifier(instance);
+    assert(identifier != null,
```

Comment:
Friendly reminder that asserts are stripped in production code.

Reply:
Thanks for the reminder. I don't see why I should delete them, but I won't be adding any in the future.

Reply:
Well this will cause debug builds to fail in a different place from production builds. If a null identifier is the result of a programming error and you want the caller to know with a well formatted error then throwing is more appropriate than an assert.  If it is safe to ignore then not crashing in debug mode is likely a better decision.